abr_analyze/gui/db_viewer.py

```diff
@@ -0,0 +1,784 @@
+# Add link to sentdex youtube
+# https://www.youtube.com/watch?v=A0gaXfM1UN0&index=2&list=PLQVvvaa0QuDclKx-QpC9wntnURXVJqLyk
+# TODO Tutorial 19 adds help button option to walk through gui
+"""
+sudo apt-get build-dep python-imaging
+sudo apt-get install libjpeg8 libjpeg62-dev libfreetype6 libfreetype6-dev
+sudo pip install Pillow
+"""
+import sys
+import tkinter as tk
+from tkinter import ttk
+
+import matplotlib
+
+matplotlib.use("TkAgg")
+from matplotlib import pyplot as plt
+from matplotlib.backends.backend_tkagg import (
+    FigureCanvasTkAgg,
+)  # , NavigationToolbar2TkAgg
+from matplotlib.figure import Figure
+import matplotlib.animation as animation
+from matplotlib import style
+import seaborn
+from PIL import Image, ImageTk
+
+import numpy as np
+from abr_analyze import DataHandler
+
+from terminaltables import AsciiTable
+
+np.set_printoptions(threshold=12)
+dat = DataHandler(db_name=sys.argv[1])
+
+
+class bcolors:
+    HEADER = "\033[95m"
+    BLUE = "\033[94m"
+    GREEN = "\033[92m"
+    YELLOW = "\033[93m"
+    RED = "\033[91m"
+    ENDC = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
+
+
+# set some constants
```

Comment:
a lot of commented out things in this section, what can be removed?

Reply:
That's my mistake, I'll go through and clean up this file. I recall adding this to the repo for the scrum presentation I did, I forgot to go back and clean it up.